frontend/src/profile/progress/ProgressUpdater.tsx

```diff
@@ -170,6 +172,8 @@ const ProgressUpdater: React.FC<ProgressUpdaterProps> = ({
                 setHours('');
                 setMinutes('');
                 request.reset();
+                // The timeline needs to know that the update request is done
+                timelineLoadMore();
```

Comment:
I'm not sure that this will do the correct thing if the user has enough timeline activity that `startKey` has a value in the useTimelineContext. If `startKey` has a value, then calling load more will start loading timeline entries from the start key, which means the newly created activity won't be included.